defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 # defaults file for openvpn
+openvpn_base_dir: ""
```

Comment:
You know, we might as well have this set to `/etc/openvpn` by default - then everywhere else instead of having `{{base_dir}}/etc/openvpn/other/path` we can just have `{{base_dir}}/other/path`.